From 2c6cc18a10dbce3c9e0b5b28ea25a3150cd2e8da Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Thu, 11 Sep 2014 11:59:41 -0400
Subject: [infiniband] mlx4: Add dependency INET

Message-id: <ab4d5b9b49f973683622b7f16139510d3cf6e036.1410425016.git.avadai@redhat.com>
Patchwork-id: 93629
O-Subject: [Patch RHEL7.1 V1 045/197] IB/mlx4: Add dependency INET
Bugzilla: 1107617 1107618 1107619 1107620
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Matan Barak <matanb@mellanox.com>

Since mlx4_ib supports IP based addressing, a dependency on INET needs
to be added, since mlx4_ib registers itself for net device events.

Signed-off-by: Matan Barak <matanb@mellanox.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit f282651de676d10e395bc7923f0087fbbba12ed7)
Signed-off-by: Amir Vadai <avadai@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/mlx4/Kconfig b/drivers/infiniband/hw/mlx4/Kconfig
index 24ab11a..fc01dea 100644
--- a/drivers/infiniband/hw/mlx4/Kconfig
+++ b/drivers/infiniband/hw/mlx4/Kconfig
@@ -1,6 +1,6 @@
 config MLX4_INFINIBAND
  tristate "Mellanox ConnectX HCA support"
- depends on NETDEVICES && ETHERNET && PCI
+ depends on NETDEVICES && ETHERNET && PCI && INET
  select NET_VENDOR_MELLANOX
  select MLX4_CORE
  ---help---
-- 
1.7.1